File: teoricos/tt11/ListaDupla/Celula.java
class Celula {
	public int elemento;
	public Celula ant;
	public Celula prox;

	public Celula() {
		this(0);
	}
		
	public Celula(int elemento) {
		this.elemento = elemento;
		ant = null;
		prox = null;	
	}

}

File: teoricos/arvores_binarias/ArvoreC/ArvoreBinariaDeLista/celula.c
#include <stdio.h>
#include <stdlib.h>
#include <celula.h>

Celula* novaCelula(char* elemento) {
   Celula* nova = (Celula*) malloc(sizeof(Celula));
   strcpy(nova->elemento, elemento);
   nova->prox = NULL;
   return nova;
}